infra/lib/ci-cluster.ts

```diff
@@ -49,17 +49,24 @@ export class CICluster extends cdk.Construct {
   }
 
   installFlux = () => {
-    const fluxChart = this.testCluster.addCdk8sChart('flux',
-      new FluxChart(
-        this.cdk8sApp, 'Flux', {}
-      )
-    );
+    const fluxNamespace = this.createNamespace(FLUX_NAMESPACE);
+    const fluxChart = this.testCluster.addHelmChart('flux2', {
```

Comment:
Thankfully, now we are using Helm, we can use `createNamespace: true` and not have to create the namespace as a dependency. This will also help with if we need to upgrade, it won't try to create the namespace each time.

Reply:
Done!